ldaptor/test/test_pureldap.py

```diff
@@ -619,7 +619,7 @@ def testToLDAP(self):
         for klass, args, kwargs, decoder, encoded in self.knownValues:
             result = klass(*args, **kwargs)
             result = str(result)
-            result = map(ord, result)
+            result = [ord(x) for x in result]
```

Comment:
Just wondering why sticking with `ord()` here and not `six.byte2int()`.

Reply:
The test seems to pass...and for this PR this is what I was aiming :)